cmd/acr/purge.go

```diff
@@ -48,15 +49,18 @@ const (
   - Delete all tags older than 1 day in the example.azurecr.io registry inside the hello-world repository, with 4 purge tasks running concurrently
 	acr purge -r example --filter "hello-world:.*" --ago 1d --concurrency 4
 	`
-	maxPoolSize              = 32 // The max number of parallel delete requests recommended by ACR server
-	manifestListContentType  = "application/vnd.docker.distribution.manifest.list.v2+json"
-	manifestIndexContentType = "application/vnd.oci.image.index.v1+json"
-	linkHeader               = "Link"
+	maxPoolSize                      = 32 // The max number of parallel delete requests recommended by ACR server
+	manifestListContentType          = "application/vnd.docker.distribution.manifest.list.v2+json"
```

Comment:
Rename contentType to MediaType which is the standard nomenclature. Specs-go should also contain some of these mediatypes directly, reference them from there.

Reply:
done.